examples/cross_sign_verify.py

```
@@ -0,0 +1,129 @@
+            ssl=ssl,
+            proxy=proxy,
+        )
+
+        if store_path and not os.path.isdir(store_path):
+            os.mkdir(store_path)
+
+        self.add_event_callback(self.message_cb, RoomMessageText)
+
+    async def login(self):
+        if os.path.exists(SESSION_DETAILS_FILE):
+            with open(SESSION_DETAILS_FILE) as f:
```

Ruff (0.12.2)

Async functions should not open files with blocking methods like `open`

(ASYNC230)